backend_new/src/passports/mfa.strategy.ts

```diff
@@ -171,9 +171,15 @@ export class MfaStrategy extends PassportStrategy(Strategy, 'mfa') {
   }
 
   async updateFailedLoginCount(count: number, userId: string): Promise<void> {
+    let lastLoginAt = undefined;
```

Comment:
Curious about this part, outside of the context of the bug. I may be misunderstanding but it seems like now (1) if a user enters the wrong credentials we delete the last login date, but it seems like that data is unrelated and still valid? Also (2) if it's the first time a user fails to login we set their last login to be now, which seems like it also isn't valid?

Reply:
Correct me if I'm wrong @YazeedLoonat, but on the `.update` function if something is passed in as undefined it is not updated.

Reply:
so @ludtkemorgan is right that `undefined` in an `.update()` or `.create()` for prisma just means "make no changes" so we won't be dumping their current last login time every time

@emilyjablonski is right though that updating the last login time to be `new Date()` also isn't totally correct, I think we should look to add "last time the user attempted to log in" and using that for comparisons to see whats going on there. 

I don't think we really use the last log in time for anything though so at least for the time being I think we can keep this change set and address this post launch of prisma

Reply:
I'll hold off on merging this in @emilyjablonski if you have other concerns lemme know, or lemme know if I'm good to merge too!

Reply:
@YazeedLoonat it's come up a handful of times in the support channel of wanting to know that data, so I think we need to keep it accurate - but if you'd rather make a ticket for a followup that's all good!

Reply:
Hey @emilyjablonski it took me a bit to figure it out, but it turns out the current implementation (typeorm) uses a similar trick. So the data that we've been providing won't change in terms of accuracy. 

So I went ahead and spun up: https://github.com/bloom-housing/bloom/issues/3824

and will move forward with merging this in